src/style.css

```diff
@@ -0,0 +1,54 @@
+html {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+.form {
+  position: relative;
+  margin: 20px 0;
+}
+
+.search {
+  width: 100%;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  padding-left: 62px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+  font-size: 16px;
+  font-family: Avenir, sans-serif;
```

Comment:
Set font styles to html tag, other elements will inherit it (except placeholders, you have to set `inherit` for them)

Reply:
Now you can use inherit